sgx/test/sgx/kmyth_sgx_test_enclave.edl

```diff
@@ -1,3 +1,47 @@
 enclave {
 	from "kmyth_enclave.edl" import *;
+	trusted {
```

Comment:
My sgx/test/sgx/kmyth_sgx_test_enclave.edl required:  

    from "sgx_pthread.edl" import *;

in order to run ```make test```.